tools/Customization/DevHome.Customization/Views/FileExplorerPage.xaml

```diff
@@ -1,21 +1,43 @@
-<Page
-    x:Class="DevHome.Customization.Views.FileExplorerPage"
-    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
-    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
-    xmlns:behaviors="using:DevHome.Common.Behaviors"
-    xmlns:views="using:DevHome.Customization.Views"
-    behaviors:NavigationViewHeaderBehavior.HeaderTemplate="{StaticResource BreadcrumbBarDataTemplate}"
-    behaviors:NavigationViewHeaderBehavior.HeaderContext="{x:Bind ViewModel}">
-
-    <Grid MaxWidth="{ThemeResource MaxPageContentWidth}" Margin="{ThemeResource ContentPageMargin}">
-        <Grid.RowDefinitions>
-            <RowDefinition Height="Auto" />
-            <RowDefinition Height="Auto" />
-            <RowDefinition Height="*" />
-        </Grid.RowDefinitions>
-
-        <ScrollView Grid.Row="2" VerticalAlignment="Top">
-            <views:FileExplorerView />
-        </ScrollView>
-    </Grid>
-</Page>
+<Page
+    x:Class="DevHome.Customization.Views.FileExplorerPage"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:behaviors="using:DevHome.Common.Behaviors"
+    xmlns:views="using:DevHome.Customization.Views"
+    behaviors:NavigationViewHeaderBehavior.HeaderTemplate="{StaticResource BreadcrumbBarDataTemplate}"
+    behaviors:NavigationViewHeaderBehavior.HeaderContext="{x:Bind ViewModel}">
+
+    <Grid MaxWidth="{ThemeResource MaxPageContentWidth}" Margin="{ThemeResource ContentPageMargin}">
+        <Grid.RowDefinitions>
+            <RowDefinition Height="Auto" />
+            <RowDefinition Height="Auto" />
+            <RowDefinition Height="*" />
+        </Grid.RowDefinitions>
+
+        <ScrollView Grid.Row="2" VerticalAlignment="Top">
+            <views:FileExplorerView />
+        </ScrollView>
+        <StackPanel x:Name="TrackRepository" Visibility="Collapsed">
+            <TextBlock Text="Enhance a Repository"/>
+            <StackPanel Orientation="Horizontal" Margin="0,20,0,20">
+                <TextBox x:Name="RootPathTextBox"
+                  x:Uid="rootPath"
+                  Header="Enter the root folder path:" PlaceholderText="Path"
+                  Width="300" HorizontalAlignment="Left"/>
+
+                <Button Content="Add" Click="AddButton_Click" Margin="10, 27, 0, 0"/>

```

Comment:
NIT: More descriptive name for the event handler.

Reply:
Thanks for the suggestion here. I will factor this in when the actual UI is implemented per design.